src/DacpacTool/DacpacTool.csproj

```diff
@@ -6,10 +6,29 @@
     <LangVersion>8.0</LangVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <SqlToolsPath>../../sqltoolsservice/src</SqlToolsPath>
+    <ManagedBatchParserPath>../../sqltoolsservice/src/Microsoft.SqlTools.ManagedBatchParser</ManagedBatchParserPath>
```

Comment:
We might want to re-use $(SqlToolsPath) here so that if we change that it will also changed the ManagedBatchParserPath.